polaris-service/src/test/java/org/apache/polaris/service/admin/PolarisOverlappingTableTest.java

```diff
@@ -0,0 +1,304 @@
+                        Catalog catalogObject =
+                            new Catalog(
+                                Catalog.TypeEnum.INTERNAL,
+                                c.catalog,
+                                propertiesBuilder.build(),
+                                System.currentTimeMillis(),
+                                System.currentTimeMillis(),
+                                1,
+                                config);
+                        try (Response response =
+                            request(EXT, "management/v1/catalogs")
+                                .post(Entity.json(new CreateCatalogRequest(catalogObject)))) {
+                          if (response.getStatus() != Response.Status.CREATED.getStatusCode()) {
+                            throw new IllegalStateException(
+                                "Failed to create catalog: " + response.readEntity(String.class));
+                          }
+                        }
+
+                        namespace = "ns";
+                        CreateNamespaceRequest createNamespaceRequest =
+                            CreateNamespaceRequest.builder()
+                                .withNamespace(Namespace.of(namespace))
+                                .build();
+                        try (Response response =
+                            request(EXT, String.format("catalog/v1/%s/namespaces", c))
+                                .post(Entity.json(createNamespaceRequest))) {
+                          if (response.getStatus() != Response.Status.OK.getStatusCode()) {
+                            throw new IllegalStateException(
+                                "Failed to create namespace: " + response.readEntity(String.class));
+                          }
+                        }
+                      });
+            });
+  }
+
+  private Response createTable(
+      DropwizardAppExtension<PolarisApplicationConfig> extension, String catalog, String location) {
+    System.out.println();
+    CreateTableRequest createTableRequest =
+        CreateTableRequest.builder()
+            .withName("table_" + UUID.randomUUID().toString())
+            .withLocation(location)
+            .withSchema(SCHEMA)
+            .build();
+    String prefix = String.format("catalog/v1/%s/namespaces/%s/tables", catalog, namespace);
+    try (Response response = request(extension, prefix).post(Entity.json(createTableRequest))) {
+      return response;
+    }
+  }
+
+  private static Invocation.Builder request(
+      DropwizardAppExtension<PolarisApplicationConfig> extension, String prefix) {
+    return extension
+        .client()
+        .target(String.format("http://localhost:%d/api/%s", extension.getLocalPort(), prefix))
+        .request("application/json")
+        .header("Authorization", "Bearer " + userToken)
+        .header(REALM_PROPERTY_KEY, realm);
+  }
+
+  private static Stream<TestConfig> getTestConfigs() {
+    return Stream.of(
```

Comment:
The two overrides here are 

BASE_EXT, Lax

And 
LAX_EXT, strict

Correct?

Reply:
Yep!